css/style.css

```diff
@@ -1,4 +1,212 @@
+  justify-content: center;
+  align-items: center;
+}
+
+.menubar__link {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, .menubar__link:active, .menubar__link:focus {
+  cursor: pointer;
+  color: #e51937;
+  box-shadow: 0 2px 0 #e51937;
+  opacity: .75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+  margin-right: 63.14px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-right: 24.86px;
+}
+
+.menubar__link:nth-child(3){
+  margin-right: 44px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-right: 46px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-right: 221.14px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-right: 30px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-right: 30px;
+}
+
+.button:nth-child(8) {
+  margin-right: 60px;
+}
+
+.button {
+  display: block;
+  background-color: #fff;
+  border: non;
+  width: 24px;
+  height: 15px;
+  position: relative;
+}
+
+.button .button__line, 
+.button .button__line:before,
+.button .button__line:after {
+  position: absolute;
+  display: block;
+  top: 50%; margin-top: -1px;
+  left: 50%; margin-left: -12px;
+  width: inherit;
+  height: 2px;
+  background-color: #e51937;
+}
+
+.button__line:before, .button__line:after {
+  content: '';
+}
+
+.button__line:before {
+  transform: translateY(-6px);
+}
+
+.button__line:after {
+  transform: translateY(6px);
+ }
+
+.button:hover {
+  cursor: pointer;
+  box-shadow: 0 0 5px #473c3c;
+  border: 1px solid #e51937;
+  transition: 0.5s;
+}
+.content {
+  position: relative;
+  background: url("../img/jpg/Tesla\ logo\ car.jpg");
+  background-repeat: no-repeat;
+}
+
+.content__main-title, .content__title {
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+  margin: 0;
+}
+
+.content__main-title {
+  font-weight: 600;
+  font-size: 90px;
+}
+
+.content__title {
+  font-weight: 500;
+  font-size: 15px;
+  padding-bottom: 26px;
+}
+
+.content__data-car {
+  display: flex;
+  padding: 480px 0px 0 321px;
+  justify-content: space-between;
+  align-items: center;
+  
+}
+
+.content__facts {
+  display: flex;
+  margin-right: 105px;
+  justify-content: space-between;
+  align-items: center;
+  width: 375p
+}
+.data__speed-icon {
+  width: 46px;
+  height: 34.13px;
+}
+
+.content__road-icon {
+  width: 30.5px;
+  height: 50px;
+}
+
+.data__speed-block {
+  padding-right: 60px;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 12px;
+}
+
+.testdrive__img {
+  width: 118px;
+  height: 22px;
+  margin-right: 40px;
+}
+
+.order-btn {
+  margin-right: 321px ;
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  transition: all 0.5s ease 0s; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)